<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel,QCheckBox,QGroupBpx
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
}

/* buttons */
QPushButton
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:rgb(134,183,200);
  border:2px solid #5F92B2;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#4dd0e1;
  border:2px solid #5F92B2;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover:pressed
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#00bcd4; 
  border:2px solid #0097a7;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item row="0" column="1">
        <widget class="QPushButton" name="addHeight">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string>增加高</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="decreaseWidth">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string>减少宽</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="resetSize">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string>重置</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="addWidth">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string>增加宽</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="decreaseHeight">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string>减少高</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="recognizeIt">
       <property name="text">
        <string>识别</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitNow">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="imgLabel">
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>camera.png</pixmap>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
